src/main/java/com/sktTask/ProductService/ProductServiceApplication.java

```diff
@@ -0,0 +1,42 @@
+package com.sktTask.ProductService;
+
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.amqp.rabbit.connection.ConnectionFactory;
+import org.springframework.amqp.rabbit.listener.SimpleMessageListenerContainer;
+import org.springframework.amqp.rabbit.listener.adapter.MessageListenerAdapter;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@SpringBootApplication
+@EnableScheduling
+public class ProductServiceApplication {
+    static final String EXCHANGE_NAME = "exchange";
+    static final String QUEUE_NAME = "queue";
+    static final String ROUTING_KEY = "product";
+
+    public static void main(String[] args) {
+        SpringApplication.run(ProductServiceApplication.class, args);
+    }
+
+    @Bean
```

Comment:
Bear in mind is better to create a `@Configuration` Class https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-configuration-classes.html